src/components/ArtifactsFilterTree/ArtifactsFilterTree.js

```diff
@@ -9,7 +9,9 @@ import './artifactsFilterTree.scss'
 
 const ArtifactFilterTree = ({ items, onChange, value, label, match, page }) => {
   const [isDropDownMenuOpen, setIsDropDownMenu] = useState(false)
-  const [filterTree, setFilterTree] = useState(value)
+  const [filterTree, setFilterTree] = useState(
+    items.find(tree => tree.id === value)?.label ?? value
```

Comment:
as I understand we need to set label and value it's id

```suggestion
    items.find(tree => tree.id === value)?.label
```